lib/DBDefs.pm.sample

```diff
@@ -292,6 +292,15 @@ sub WEB_SERVER                { "www.musicbrainz.example.com" }
 #     return \%CACHE_MANAGER_OPTIONS
 # }
 
+# Sets the TTL for entities stored in memcached, in seconds. On slave servers,
+# this is set to 1 hour by default, to mitigate MBS-8726. On standalone
+# servers, this is set to 0 (meaning no expiration is set), because cache
+# invalidation is already handled by the server in that case.
+# sub ENTITY_CACHE_TTL {
+#     return 3600 if shift->REPLICATION_TYPE == RT_SLAVE;
+#     return 0;
+# }
```

Comment:
I think we should have a simpler example method here in the sample file, e.g. `# sub ENTITY_CACHE_TTL { 0 }`. The more complex slave/non-slave logic can stay in `DBDefs::Default`.